doc/script_commands.txt

```diff
@@ -3582,6 +3582,32 @@ This command does not count skills which are set as flag 4 (permament granted) (
 
 ---------------------------------------
 
+*getrandmobid(<type>{,<flag>{,<level>}})
+
+This command return a random monster ID from the random monster group.
+If <level> is provided it will check against the monster level.
+It will return 0 if invalid.
```

Comment:
```suggestion
This command returns a random monster ID from the random monster group.
If <level> is provided it will check against the monster level.
Returns 0 if invalid.
```